src/pip/_internal/cli/spinners.py

```diff
@@ -81,55 +75,47 @@ def finish(self, final_status):
 # act as a keep-alive for systems like Travis-CI that take lack-of-output as
 # an indication that a task has frozen.
 class NonInteractiveSpinner(SpinnerInterface):
-    def __init__(self, message, min_update_interval_seconds=60):
-        # type: (str, float) -> None
+    def __init__(self, message: str, min_update_interval_seconds: float = 60) -> None:
```

Comment:
```suggestion
    def __init__(self, message: str, min_update_interval_seconds: float = 60.0) -> None:
```

I’m surprised Mypy didn’t complain.

Reply:
Well, maybe I had a mistake. But that's true, Mypy had to say something. But I think its just a bug with Mypy.

Reply:
60 is compatible with float, so I'm not surprised this is OK.

The following test file typechecks as fine, and I think that's reasonable:

```python
def f(n: float) -> None:
    print(n)

def g(n: int) -> None:
    f(n)

f(12.0)
f(12)
g(12)
```

If I add a call `g(12.0)` I get

```
test.py:10: error: Argument 1 to "g" has incompatible type "float"; expected "int"
Found 1 error in 1 file (checked 1 source file)
```

Reply:
I see. Thank you. However, we have to resolve the issue of the previous review (_the PEP 661 application_). Maybe we are using and old Mypy.

Reply:
Taking the @pfmoore [commentary](https://github.com/pypa/pip/pull/10065#discussion_r652696493), this is not exactly a problem, isn't it? Now I'll resolve this conversation.